src/System.Net.NameResolution/src/System.Net.NameResolution.csproj

```diff
@@ -216,6 +222,11 @@
     </Compile>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetGroup)' == 'net46'">
```

Comment:
We should actually add some notes about this in the repo documentation as opposed to everywhere they are used in the repo because they are used in a lot of places.